pkg/hub/manager.go

```diff
@@ -149,7 +149,11 @@ func RunControllerManager(ctx context.Context, controllerContext *controllercmd.
 			controllerContext.EventRecorder,
 		)
 	}
-
+	globalManagedClusterSetController := managedclusterset.NewGlobalManagedClusterSetController(
```

Comment:
I think this should be in the same feature gate of defaultclusterset. And please also update the description of that feature gate.